src/components/RoomForm/RoomCreate/RoomCreate.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useRef } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import '../RoomForm.css';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { roomCreate, setRoomUserName, clearRoomData } from '../../../reducers/RoomSlice';
+import { clearChatHistory } from '../../../reducers/ChatroomSlice';
+import store from '../../../store';
+
+
+function RoomCreate() {
+  const dispatch = useDispatch();
+  const signedIn = useSelector(state => (state.user.userId !== null));
```

Comment:
It would be awkward if we created a new state variable like `isSignedIn` and it became out of sync with `userId`. If `userId` was `null` but `signedIn` was `true`, which would be the better source of truth? Thus, I implemented it the way it currently is.

As mentioned before, ideally we would have a session token or something of the sort that would be more reliable to determine if the user is signed in.